esrally/track/track.py

```diff
@@ -564,15 +565,20 @@ def admin_op(self):
         # pylint: disable=comparison-with-callable
         return self.value > 1000
 
+    def to_hyphenated_string(self):
+        # Pylint complains that self.name is not iterable
+        # pylint: disable=not-an-iterable
+        return "".join(["-" + c.lower() if c.isupper() else c for c in self.name]).lstrip("-")
```

Comment:
nit: took me a little to understand (by reading the method name) that we are hyphenating based on capitalization. Looking at the implementation it became clear, so not sure we need to rename, but perhaps a docstring would be useful?

Reply:
I added a docstring in https://github.com/elastic/rally/pull/1112/commits/fc29cc6685bd18e77824ae00eacbdad9acadb411.

Reply:
Very clear thanks!